subprojects/test-runner/test-inhouse-runner/src/main/kotlin/com/avito/android/rule/ActivityScenarioRule.kt

```diff
@@ -9,15 +9,20 @@ import com.avito.android.rule.base.BaseActivityScenarioRule
  * actions that send out intents which need to be verified or stubbed. This is similar to
  * MockitoAnnotations.initMocks.
  */
-class ActivityScenarioRule<T : Activity>(activityClass: Class<T>) : BaseActivityScenarioRule<T>(
+class ActivityScenarioRule<T : Activity>(
+    activityClass: Class<T>,
+    private val stubIntents: Boolean
+) : BaseActivityScenarioRule<T>(
     activityClass = activityClass,
     initialTouchMode = true,
     launchActivity = false,
 ) {
 
     override fun afterActivityLaunched() {
         Intents.init()
-        com.avito.android.test.Intents.stubEverything()
+        if (stubIntents) {
+            com.avito.android.test.Intents.stubEverything()
```

Comment:
Could we write a stubber that will write logs with what it stubbed? It will help If someone use stub but forget about that behaviour

Reply:
I disabled the stubbing by default. Now it should be more clear for end-users. I agree that logs will be very useful here, we can add them later.

Reply:
Added logs